### PR TITLE
Clarifying

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1867

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
@dariomws please have a look, thanks

**Comment by get-itips** [comment]
@JohanFreelancer9 ready for final review

**Review by dariomws — APPROVED** [review]

File: Exchange/ExchangeServer2013/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help.md [hunk 1]
```diff
@@ -184,14 +184,14 @@ You must also define the external Autodiscover endpoint for your on-premises org
 > [!NOTE]
 > You can use the [Get-IntraOrganizationConfiguration](https://docs.microsoft.com/powershell/module/exchange/Get-IntraOrganizationConfiguration) cmdlet in both your on-premises and Microsoft 365 or Office 365 tenants to determine the endpoint values needed by [New-IntraOrganizationConnector](https://docs.microsoft.com/powershell/module/exchange/New-IntraOrganizationConnector) cmdlet.
 
-Using Windows PowerShell, run the following cmdlet:
+Using Windows PowerShell, run the following cmdlet, remember to replace the UserPrincipalName:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help.md:187]:
```suggestion
Using Windows PowerShell, run the following cmdlet and remember to replace the UserPrincipalName:
```

File: Exchange/ExchangeServer2013/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help.md [hunk 2]
```diff
@@ -184,14 +184,14 @@ You must also define the external Autodiscover endpoint for your on-premises org
...
 
 ```powershell
-$UserCredential = Get-Credential
-$Session = New-PSSession -ConfigurationName Microsoft.Exchange -ConnectionUri https://outlook.office365.com/powershell-liveid/ -Credential $UserCredential -Authentication Basic -AllowRedirection
-Import-PSSession $Session
+Connect-ExchangeOnline -UserPrincipalName navin@contoso.com -ShowProgress $true
 New-IntraOrganizationConnector -name ExchangeHybridOnlineToOnPremises -DiscoveryEndpoint <your on-premises Autodiscover endpoint> -TargetAddressDomains <your on-premises SMTP domain>
 ```
+> [!NOTE]
+> The above step uses Exchange Online PowerShell V2 module, for more information, see [Connect to Exchange Online PowerShell](https://docs.microsoft.com/powershell/exchange/connect-to-exchange-online-powershell).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help.md:194]:
```suggestion
> The above step uses the Exchange Online PowerShell V2 module. For more information, see [Connect to Exchange Online PowerShell](https://docs.microsoft.com/powershell/exchange/connect-to-exchange-online-powershell).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor edits.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**